_posts/2022-02-01-Fortran-Newsletter-February-2022.md

```diff
@@ -0,0 +1,187 @@
+  Get user name and email using got config if available else use defaults
+* [#648](https://github.com/fortran-lang/fpm/pull/648) (WIP):
+  Add note about relocation of manifest reference
+* [#622](https://github.com/fortran-lang/fpm/pull/622) (WIP):
+  Cleanup the backend output
+* [#608](https://github.com/fortran-lang/fpm/pull/608) (WIP):
+  --env switch lets you specify the prefix of the compiler-related environment variables
+* [#539](https://github.com/fortran-lang/fpm/pull/539) (WIP):
+  Add parent packages into dependency tree
+* [#498](https://github.com/fortran-lang/fpm/pull/498) (WIP):
+  Compiler flags profiles
+
+`fpm` is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://fpm.fortran-lang.org/en/tutorial)
+to learn how to build your package with fpm, and the [manifest reference](https://fpm.fortran-lang.org/en/spec/manifest.html)
+to learn what are all the things that you can specify in the fpm.toml file.
+
+* Browse existing *fpm* packages on the [fortran-lang website](https://fortran-lang.org/packages/fpm)
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Flang
+
+Recent development updates:
+
+TO DO
+
+Call notes are recorded and available upon request [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY). Please contact Alexis Perry-Holby at aperry@lanl.gov for document access.
+
+### LFortran
+
+TO DO
+
+
+We are looking for new contributors. Please do not hesitate to contact us if
+you are interested. We will help you get up to speed.
+
+## Events
+
+* fpm has a new documentation website hosted at
+  [fpm.fortran-lang.org](https://fpm.fortran-lang.org/).
+  This website will provide user-oriented tutorials and how-to guides, as well
+  as developer-oriented reference documents and specifications.
+  We welcome all contributions to the fpm documentation, including translations
+  to other languages.
+  Please visit the [fpm-docs repo](https://github.com/fortran-lang/fpm-docs) to
+  get started.
+* We had our 22st Fortran Monthly call on January 17.
+  You can watch the recording below:
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/NSvL2yrefH8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
@milancurcic I couldn't find the recording on Youtube. Is it already there?

Reply:
Thanks for the reminder, I'll do it today and update the URL.